src/cmd/serve.rs

```diff
@@ -117,7 +98,6 @@ pub fn execute(args: &ArgMatches) -> Result<()> {
         info!("Building book...");
 
         // FIXME: This area is really ugly because we need to re-set livereload :(
```

Comment:
Is this still applicable? Do we need to re-set this again?

Reply:
Yes. The book is loaded from scratch (and thus the config is reloaded from scratch), and it needs to inject the URL into the output.